minepi/skin.py

```diff
@@ -40,8 +40,7 @@ def __init__(
         self._skin: Optional[Image.Image] = None
         self._head: Optional[Image.Image] = None
 
-        if raw_cape is not None:
-            self.set_cape(raw_cape)
+        self.set_cape(raw_cape)
```

Comment:
I cannot imagine that being helpful since you removed the check for capes?
Are you sure that you have the proper version here? Since I just added the check to prevent this exact error from happening. 
And when comparing the line shown in the error to the source, it does not match.

Reply:
I think there was a version issue previously where the latest version of MinePI was different on github vs pypi. If this is fixed now, can disregard.